codeforboston/maple
# Bootstrap Migration for Learn Testimony Pages

# Summary

This PR covers a first pass at the Bootstrap migration of the Learn Testimony pages. This includes:
- [ ] Conversion to Typescript
- [ ] Removing CSS Module styles in favor of innate Bootstrap styles and styled components

This addresses the issue tracked in #1443  

There are some minor style changes here that I don't think should be a deal breaker, and I ran into some font sizing inconsistency of the sort we're likely interested in addressing in the dev/design sync.

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.

# Screenshots

`The Basics of Testimony` and `Role of Testimony` should be identical to prod. The style differences are mainly around the text styles on the `Writing Effective Testimony` and `The MA Legislative Process` pages - it'd be helpful to pick a consistent font/font size here to just use on all these cards for readability and reusability.

Old `Writing Effective Testimony` card:
<img width="867" alt="Screenshot 2024-02-27 at 2 30 46 PM" src="https://github.com/codeforboston/maple/assets/2694761/015aa2bb-3260-41eb-a7a6-44ee55564541">

Updated `Writing Effective Testimony` card:
<img width="846" alt="Screenshot 2024-02-27 at 2 30 52 PM" src="https://github.com/codeforboston/maple/assets/2694761/b0ceb61a-515f-4c47-a5c4-a321ee9def9e">

Old `The MA Legislative Process` card:
<img width="1059" alt="Screenshot 2024-02-27 at 2 31 13 PM" src="https://github.com/codeforboston/maple/assets/2694761/9de89907-199d-47d5-b728-6b22e8cb4bbe">

Updated `The MA Legislative Process` card:
<img width="1103" alt="Screenshot 2024-02-27 at 2 31 17 PM" src="https://github.com/codeforboston/maple/assets/2694761/a1e37f14-aaf7-40e4-bb93-f342ff48d749">

# Known issues

_If you've run against limitations or caveats, include them here. Include follow-up issues as well._

# Steps to test/reproduce
1. Navigate to the Learn About Testimony page
2. Check each of the tabs to see that they display the expected content and styles (including resizing the window to check responsive styles).
